article/Name-YEAR.md

```diff
@@ -151,36 +151,40 @@ analysis of the results.
 ### Deterministic model
 
 The reproduced population dynamics for the host population and for the
-specialist parasitoids (Figures 3a and 3b) are very similar to the ones in
+specialist parasitoids (@fig:figure3 (a) and (b)) are very similar to the ones in
 Hassell's paper. When the level of clumping of parasitoid attacks is high, both
 populations decrease during the first 10 generations before they stabilise until
 the end of the simulation, with the host population twice as big as the
 specialist parasitoid population. When the extent of clumping is lower, both
 populations show decreasing oscillations during the 50 simulated generations.
 The difference between this article and the original publication resides in the
-@fig:figure3 and Figure 3d, where we standardized the axes of the graphs. At first
+@fig:figure3 (c) and (d), where we standardized the axes of the graphs. At first
 sight, in Hassell's paper, the k-values in Figure 3c seem almost as high as in
 Figure 3d. However, when using the same scale for the two graphs, we can see
 that the k-values in the case of high clumping show much less variation than
 when the lower clumping is used.
 
 The simulations performed to study the relationship between the population
 dynamics of the hosts and the generalist parasitoids are also very similar to
-the ones done by Hassell. This show a density dependant relationship between the
+the ones done by Hassell (@fig:figure4). This shows a density dependant relationship between the
 two populations, where the natural enemy regulates the host population until
 they both reach a stable equilibrium. This is also the conclusion when we
-observe the relation between the two populations in Figures 3a and 3c.
+observe the relation between the two populations in @fig:figure3 (a) and (c).
 
 ### Stochastic model
 
+When the models include stochasticity, the resulting population dynamics between hosts and parasites are much more different than when the models are deterministic. Overall, the results obtained in the replications are pretty similar than the ones from Hassell, except for @fig:figure6 (d), where the correlation between the mortality and the host density is not nearly as strong as in the original paper. Whether we look at the model for the specialist parasitoids or the one for the generalist parasitoids, both show irregular oscillations in the host-parasitoid population dynamics (@fig:figure5 and @fig:figure6, (a), (b) and (c)). The addition of stochastic parameters prevents the stabilization of host populations and makes it more difficult to identify parasitoids as a density-dependent control factor. The relationship between the k-values and the host density are similar in Hassell's publication and in ours (except in @fig:figure6 (d)). The simulations done in this paper tend to have a determination coefficient (R^2) a little higher than the one found by Hassell, but they are generally really close.
```

Comment:
je sais pas si ça se dit, et/ou je trouve que c'est drolement dit cette phrase:
"The relationship between the k-values and the host density are similar in Hassell's publication and in ours"
Quelque chose comme 
"Hassell found a similar relationship between the k-value and the host density to ours"

Reply:
ou quelque chose du genre